account.go

```diff
@@ -30,8 +30,17 @@ type account interface {
 	Call(ctx context.Context, call types.FunctionCall) ([]string, error)
 	Nonce(ctx context.Context) (*big.Int, error)
 	EstimateFee(ctx context.Context, calls []types.FunctionCall, details types.ExecuteDetails) (*types.FeeEstimate, error)
-	Execute(ctx context.Context, calls []types.FunctionCall, details types.ExecuteDetails) (*types.AddInvokeTransactionOutput, error)
-	Declare(ctx context.Context, classHash string, contract rpcv02.ContractClass, details types.ExecuteDetails) (types.AddDeclareResponse, error)
+	Execute(
+		ctx context.Context,
+		calls []types.FunctionCall,
+		details types.ExecuteDetails,
+	) (*types.AddInvokeTransactionOutput, error)
```

Comment:
This should be on a single line like the other other functions for consistency.

Reply:
The same goes for other places where the function signature is spread across multiple lines.

Reply:
Got it.
I was trying some new plugins for my editor and seems like it formatted stuff in a weird way.
I think the same happen in other parts of the code.